libs/parse/sections/base.py

```diff
@@ -24,17 +28,18 @@ def __init__(self, base, full_header, header, content, silent):
         self.base = base
         if base:
             self.title = base.title
-        self.full_header = full_header
-        self.header = header
-        self.content = content
-        self.silent = silent
+        self.full_header: str = full_header
+        self.header: str = header
+        self.content: str = content
+        self.silent: bool = silent
 
-        self.is_parsing = False
-        self.parsed = False
+        self.is_parsing: bool = False
+        self.parsed: bool = False
         self._key = None
         self._top = None
-        self._sub_sections = None
-        self._old_content = content
+        self._sub_sections: Dict[str, BaseSection] = None  # TODO: fix None
```

Comment:
Before merging, pls, cgange None to Dict if possible.

Reply:
What do you mean? You just don't like default `None` value for `Dict` type? :)

Reply:
Yes. I though it would be better way if we strict-type it. I am Javist after all =D

Reply:
So, do you agree that we can leave it as-is for now? _(like with `_sub_data` below)_